internal/version/version.go

```diff
@@ -33,3 +35,6 @@ func Write(ctx context.Context, store storage.Store, txn storage.Transaction) er
 
 	return nil
 }
+
+// UserAgent defines the current OPA instances User-Agent default header value
```

Comment:
Mega nit: I like to use punctuation on exported symbol comments because the sentence/paragraph shows up in GoDoc.

Reply:
Mega nit?! 😮Noted, and will fix it before we cut the release

Reply:
Fixed in https://github.com/open-policy-agent/opa/pull/1583